src/core/openthread-core-default-config.h

```diff
@@ -1537,4 +1537,18 @@
 #define OPENTHREAD_CONFIG_DISABLE_CCA_ON_LAST_ATTEMPT 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_INTERRUPT_CONTEXT_AES
+ *
+ * Define as 1 to enable AES usage in interrupt context.
+ *
+ * @note If this feature is enabled, a software AES under platform layer must be implemented when:
+ *       1. The platform's hardware AES only supports single AES engine;
+ *       2. The platform's hardware AES doesn't support interrupt context.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_INTERRUPT_CONTEXT_AES
```

Comment:
Given that this no longer applies to any code in OpenThread core, we should consider moving this into a platform-specific config.

Reply:
Done, thanks.